app/controllers/tmdb_controller.rb

```diff
@@ -51,27 +52,29 @@ def full_cast
     end
   end
 
-  # single actor search
+  # movies for single actor
   def actor_search
     if params[:actor].present?
       @results = Tmdb::Client.get_movies_for_actor(
         actor_name: params[:actor],
-        page: (params[:page].presence || 1),
-        sort_by: (params[:sort_by].presence || 'popularity')
+        page: params[:page],
+        sort_by: params[:sort_by]
```

Comment:
I moved this logic into the client. I figured the query should know how to handle itself when it comes to defaults.